mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/model/impl/MessageParser.java

```diff
@@ -124,7 +126,7 @@ private ParsedAttachment retrieveAttachment(Entity entity) throws IOException {
         Optional<ContentTypeField> contentTypeField = getContentTypeField(entity);
         Optional<ContentDispositionField> contentDispositionField = getContentDispositionField(entity);
         Optional<ContentType> contentType = contentTypeField.map(ContentTypeField::getBody)
-            .filter(string -> !string.isEmpty())
+            .filter(Predicate.not(Strings::isNullOrEmpty))
```

Comment:
also here?

Reply:
it can be null

Reply:
that's my point. If it is null, before this was throwing, now it's passing. What is expected? Is it tested?

For me it's no more a refactoring in this case, it's a bug fix, and it needs a test.